### DISCUSSION TITLE
Wrong dependency in commit message

### DISCUSSION THREAD

**Discussion by nyg** [renovatebot/renovate#14319] [Request Help] [answered]
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

As can be seen in this commit https://github.com/nyg/dashborg/commit/ba74ba28dbe750813a5de5632d0d5c862f97311f, it says that path-parse was update to version 1.0.7 but instead we can see in the diff that it is the jszip dependency that was downgraded to version 3.0.6 from 3.0.7.

Commit history:
https://github.com/nyg/dashborg/commit/1a7178eaebab07c04f9017fea259d703cd44a197 (12:42)
https://github.com/nyg/dashborg/commit/78b301a0f8392603f0906f6adb1ddd03f0f193d7 (12:44)
https://github.com/nyg/dashborg/commit/ba74ba28dbe750813a5de5632d0d5c862f97311f (14:27, concerned commit)

### Relevant debug logs

<details><summary>Logs</summary>

```
Logs not kept, issue happened more than 3 days ago…
```

</details>

### Have you created a minimal reproduction repository?

No reproduction, but I have linked to a public repo where it occurs

**Comment by rarkins** [comment, marked as answer]
This was fixed by #14173

You can close that PR. If you're interested to remediate it, either refresh your lock file manually or enable lock file maintenance in Renovate
